assets/prototype/domain/eventEditor/datetimes/dateList/DateList.js

```diff
@@ -21,20 +20,9 @@ const btnRowStyle = {
 	width: '100%',
 };
 
-const DateList = ({ datetimes, error, loading, tickets }) => {
-	if (!loading) {
-		console.log('%c DateList', 'color: orangered; font-size: 14px;');
-		console.log('%c > datetimes:', 'color: tomato;', datetimes);
-		console.log('%c > loading:', 'color: tomato;', loading);
-	} else if (error) {
-		console.log('%c > error:', 'color: red; font-size:16px;', error);
-	}
+const DateList = ({ datetimes, tickets }) => {
 	const header = <H3 style={{ margin: '2rem 0 .5rem' }}>{'Dates List'}</H3>;
 
-	if (loading) return <LoadingIndicator header={header} message='loading dates...' />;
-
-	if (error) return <ErrorIndicator error={error} header={header} />;
-
```

Comment:
I was wrongly assuming that this PR will prevent this noise: https://github.com/eventespresso/event-espresso-core/pull/1974

Reply:
have reverted back changes, but anyway I see that we have the same duplication in `TicketList`;
if it's worth, we can have a small HOC to avoid duplication